app/assets/javascripts/maps.es6

```diff
@@ -148,19 +148,23 @@ window.COPO.maps = {
   },
 
   buildMarkerPopup(checkin) {
+    let address = checkin.city;
+    if(checkin.address){
+      address = COPO.utility.commaToNewline(checkin.address)
+    }
```

Comment:
If checkin is fogged, has no address, only city. So this allows the marker to show full address if available, city if not.